usl_pipeline/cloud_functions/main.py

```diff
@@ -40,12 +41,28 @@ def build_feature_matrix(cloud_event: functions_framework.CloudEvent) -> None:
     another GCP bucket for eventual use in model training and prediction.
     """
     logging.basicConfig(level=logging.WARN)
+
+    # To handle retries, check the time created and retry until the event is too old.
+    # https://cloud.google.com/functions/docs/bestpractices/retries
+    event_time = datetime.datetime.fromisoformat(cloud_event.data["timeCreated"])
+    event_age = (
+        datetime.datetime.now(datetime.timezone.utc) - event_time
+    ).total_seconds()
+    max_age = 20
```

Comment:
Can you make it a top level constant to make it easier to modify later if needed?

Reply:
Done & added some more comments explaining how this relates to retries.